script/publish-docs

```diff
@@ -14,9 +14,9 @@ npm run doc
 [ -d tmp/website ] || {
   REPO=https://github.com/probot/probot.github.io
 
-  # If GH_TOKEN environment variable is present, use that to push.
-  if [[ -n "$GH_TOKEN" ]]; then
-    REPO=${REPO/https:\/\//https://${GH_TOKEN}@}
+  # If OCTOKITBOT_PAT environment variable is present, use that to push.
```

Comment:
According to the workflow file, this should still be `GITHUB_TOKEN` - so either this should change, or the workflow file should be `OCTOKITBOT_PAT`

Reply:
ooopsie

Reply:
fixed in ce93c79